### ISSUE TITLE
Allow to decorate class methods

### ISSUE THREAD

**Issue by AKuederle** [AKuederle/funcmap#4]
The tool should be able to wrap class methods

**Comment by AKuederle** [comment]
This is already possible if you map in the init function (not using a decorator):

````
import funcmap

class Test:

    def __init__(self):
        self.mapper = funcmap.FuncMapper()
        self.mapper.map(r'test')(self.test)

    def test(self):
        print('test')

    def handle_message(self, msg):
        return self.mapper(msg)

Test().mapper('test')
`````

Maybe the syntax could be further improved by allowing direct accsess to the map dictionary like:
```
self.mapper[r'test'] = self.test
```

**Comment by AKuederle** [comment]
Other option would be to allow a second argument for `self.mapper.map(regex, func=None)` to either use it as decorator or directly to create the mapping

**Comment by AKuederle** [comment]
Solution implmented in a5bdb007be3a79499d278bfb525abaf8e30f8a57

**Comment by AKuederle** [comment]
Available in 1.0.2
